FrontEnd/src/components/profileList/Loader.jsx

```diff
@@ -0,0 +1,36 @@
+import css from "./Loader.module.css";
+import { Carousel, CarouselProps, ConfigProvider } from "antd";
+
+export default function Loader() {
+  return (
+    <div className={css.container}>
+      <ConfigProvider
+        theme={{
+          components: {
+            Carousel: {
+              colorBgContainer: "#40af85",
+              dotActiveWidth: 32,
+              dotWidth: 32,
+              dotHeight: 6,
+              lineHeight: 1
+            },
+          },
+        }}
+      >
+        <Carousel autoplay autoplaySpeed={1000} infinite dotPosition="top" >
```

Comment:
https://github.com/ita-social-projects/Forum/pull/244/files#r1341494108

Reply:
Done in PR #244. This page will be removed as soon as that PR merged and replaced with that one accordingly right away.